player

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include <vector>
+
+class Player{
+    std::vector<int> numbers;
+    int score;
+    bool is_pc;
```

Comment:
I don't like this boolean in costructor.

Could you make enum {
HUMAN,
COMPUTER
}

In contructor have enum parametr